package/MDAnalysis/analysis/dihedrals.py

```diff
@@ -265,12 +265,21 @@ class Ramachandran(AnalysisBase):
     """
 
     def __init__(self, atomgroup, c_name='C', n_name='N', ca_name='CA',
-                 **kwargs):
+                 check_protein=True, **kwargs):
         """Parameters
         ----------
         atomgroup : AtomGroup or ResidueGroup
             atoms for residues for which :math:`\phi` and :math:`\psi` are
             calculated
+        c_name: str (optional)
+            name for the backbone C atom
```

Comment:
specify defaults for these in the docstring

Reply:
and maybe add a line showing how they can be used

Reply:
I remember discussing this with @orbeckst and coming to the agreement that defaults are in the function signature anyway, so it's easier for docs not to diverge if they're not specified in the docstring. Happy to go either way though